network/src/validator_network/consensus.rs

```diff
@@ -100,13 +101,21 @@ impl ConsensusNetworkSender {
         &mut self,
         recipient: PeerId,
         message: ConsensusMsg,
+    ) -> Result<(), NetworkError> {
+        self.send_raw(recipient, message.to_bytes().unwrap()).await
+    }
+
+    pub async fn send_raw(
```

Comment:
This is a nice find and fix to avoid repeated serialization.

Maybe send_bytes() would be a better name?

Please add a comment here to describe this function.